homeassistant/components/updater/binary_sensor.py

```diff
@@ -26,11 +26,14 @@ class UpdaterBinary(CoordinatorEntity, BinarySensorEntity):
     _attr_unique_id = "updater"
 
     @property
-    def is_on(self) -> bool | None:
-        """Return true if the binary sensor is on."""
-        if not self.coordinator.data:
-            return None
-        return self.coordinator.data.update_available
+    def available(self) -> bool:
+        """Return if entity is available."""
+        return True
```

Comment:
Is the problem that we schedule the first coordinator refresh and the load the platform with the entity. So the entity may not receive an update until after 24 hours after start.